docs/learn/smart-contract-developers/create-nft-collection-with-lsp7-tokenId.md

```diff
@@ -0,0 +1,953 @@
+---
+sidebar_label: '🃏 Create NFT Collection with a contract address as tokenId'
+description: 'Learn how to create a NFT collection with a contract address as tokenId'
+sidebar_position: 8
+---
+
+# Create NFT Collection with a contract address as tokenId
+
+:::note
+
+This guide builds on top of a Hardhat project using TypeScript as described in the [Getting Started](./getting-started.md) section.
+
+:::
+
+## Introduction
+
+If you have been looking at our [LSP8 token standard](../../standards/tokens/LSP8-Identifiable-Digital-Asset.md), you probably have noticed that addresses could be used as `tokenIds`. In this guide, we will replicate a real-world example where this could be useful.
+
+## Use Case
+
+Let's take the example of [The Dematerialised](https://thedematerialised.com), and more precisely their KLxENDLESS PHYSITAL COLLECTION. As you can see in the picture below, we have three medaillons of different colors.
+
+![KLxENDLESS PHYSITAL COLLECTION](../../../static/img/learn/klxendless-physital-collection.png)
+
+Each medaillon has a supply limit and each medaillon collection has its own metadata with different attributes. What we will do is create an [LSP8 collection](../../standards/tokens//LSP8-Identifiable-Digital-Asset.md) that will represent the KLxENDLESS PHYSITAL collection and each medaillon a sub-collection (purple, blue or gold) will be represented by a limited supply [LSP7 token](../../standards/tokens//LSP7-Digital-Asset.md).
+
+## Prerequisites
+
+- [Hardhat installed and initialized](./getting-started.md)
+- [`@lukso/lsp-smart-contracts`](https://www.npmjs.com/package/@lukso/lsp-smart-contracts) package installed from npm using `npm i @lukso/lsp-smart-contracts@0.14.0` (or the latest version)
+- [@erc725/erc725.js](https://www.npmjs.com/package/@erc725/erc725.js) package installed from npm using `npm i @erc725/erc725js` (we will use it to encode the token metadata)
```

Comment:
either full with or without quotes

<img width="316" alt="image" src="https://github.com/lukso-network/docs/assets/477945/500c119b-5975-40f8-aaa0-68fb326fe1c1">

but idk why one must be written like `this` and the other like that

Reply:
im not sure need to ask copilot